server/backend/api/app/crud/crud_access_control_privilege.py

```diff
@@ -8,40 +8,62 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def get_delegated_admin_by_user_and_app_id(
+def is_delegated_admin_by_app_id(
     db: Session, user_id: int, application_id: int
-) -> FamAccessControlPrivilege:
+) -> bool:
     """
     Find out from `app_fam.fam_access_control_privilege` if user is the delegated admin of the application.
 
     :param user_id: primary id that is associated with the user.
     :param application_id: primary id that is associated with the application.
-    :return: FamAccessControlPrivilege record, role information the user is able to manage
+    :return: true if there is at least one record, false if no record
     """
     return (
-        db.query(FamAccessControlPrivilege)
+        True
+        if db.query(FamAccessControlPrivilege)
         .join(FamRole)
         .filter(
             FamAccessControlPrivilege.user_id == user_id,
             FamRole.application_id == application_id,
         )
-        .one_or_none()
+        .first()
```

Comment:
so it will return the first row found or none, we only need to know the user has at least one record, then means the user is the delegated admin of the application